tgui/packages/tgui/interfaces/Photocopier.js

```diff
@@ -33,7 +44,7 @@ export const Photocopier = (props, context) => {
               No inserted item.
             </Box>
           </Section>
-        )}
+        )},
```

Comment:
These commas seem to only clutter the UI (they are rendered as text, yes, they can be safely omitted).